BETA.md

```diff
@@ -32,14 +32,24 @@ From the repo root run `sbt publishLocal`
 These steps are required on each project.
 
 ## Step 1 - add semanticdb-scalac compiler plugin to your project
-Some features of the server rely on the artifacts produced by [`semanticdb-scalac`](http://scalameta.org/tutorial/#semanticdb-scalac).
-Enable it in your sbt build by:
-
-- adding the setting
-
-    `addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)`
-
-- adding the `-Yrangepos` scalac flag (e.g. `scalacOptions += "-Yrangepos"`)
+Some features of the server rely on the artifacts produced by the [`semanticdb-scalac`](http://scalameta.org/tutorial/#semanticdb-scalac) compiler plugin.
+There are two alternative ways to install `semanticdb-scalac`.
+
+The first option is to enable `semanticdb-scalac` permanently for your project in `build.sbt` with:
+
+```scala
+libraryDependencies += compilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)
+scalacOptions += "-Yrangepos"
+```
+
+The second option is to enable `semanticdb-scalac` only for an active sbt session by running `semanticdbEnable` from the sbt shell.
```

Comment:
I was exactly looking for this, but can this be enable without touching any of the `sbt` files?

Reply:
Yes, this works without touching any sbt files.